src/wallet/receive.cpp

```diff
@@ -223,7 +223,8 @@ CAmount CachedTxGetAvailableCredit(const CWallet& wallet, const CWalletTx& wtx,
 
 void CachedTxGetAmounts(const CWallet& wallet, const CWalletTx& wtx,
                   std::list<COutputEntry>& listReceived,
-                  std::list<COutputEntry>& listSent, CAmount& nFee, const isminefilter& filter)
+                  std::list<COutputEntry>& listSent, CAmount& nFee, const isminefilter& filter,
+                  bool report_change)
```

Comment:
In 812c45af8ed06c985cb5349b11043952e013003d "rpc: add a report_change parameter to listsinceblock"

nit: s/report_change/include_change for consistency with the rpc parameter.

Reply:
Done.